Podfile

```diff
@@ -23,7 +23,8 @@ workspace 'WooCommerce.xcworkspace'
 ## =====================================
 ##
 def aztec
-  pod 'WordPress-Editor-iOS', '~> 1.19.9'
+#  pod 'WordPress-Editor-iOS', '~> 1.19.9'
```

Comment:
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/woocommerce/woocommerce-ios/blob/3ae3a5353a6e6cbd4e24dbb7a523847381ba19df/Podfile#L26"></span>Layout/CommentIndentation: Incorrect indentation detected (column 0 instead of 2).</td>
    </tr>
  </tbody>
</table>